src/Veldrid.VirtualReality/OpenVR/OpenVRContext.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System;
-using System.Numerics;
 using System.Text;
+using System.Numerics;
```

Comment:
nit: This seems like an unnecessary change (and it's less alphabetical than before).